ogpf.f90

```diff
@@ -1733,37 +1733,37 @@ subroutine processcmd(this)
         ! write the plot style for data
         ! this is used only when 3D plots (splot, cplot) is used
         if (allocated(this%txtdatastyle)) then
-            write ( this%file_unit, '(a)' ) 'set style data '//this%txtdatastyle  !set data style
-            write ( this%file_unit, '(a)' )  ! emptyline
+            write ( this%file_unit, '("set style data ", a)' ) this%txtdatastyle
+            write ( this%file_unit, '(a)' )

```

Comment:
Please use ' instead " for consistency
Please keep inline comments

Reply:
This can't be done the usage of double quotes here is to escape the outer single quote. About the comments they aren't really useful in my opinion since it's clear the intention of those statements, we know Fortran, right?